src/devices/Adxl345/README.md

```diff
@@ -13,10 +13,10 @@ SpiConnectionSettings settings = new SpiConnectionSettings(0, 0)
 };
 
 // get SpiDevice(In Linux)
-UnixSpiDevice device = new UnixSpiDevice(settings);
+var device = SpiDevice.Create(settings);
```

Comment:
remove the comments and the second call since they are the same now

Reply:
Updated